contributing/DEVELOPING.md

```diff
@@ -84,15 +85,14 @@ For any public AMP document like: http://output.jsbin.com/pegizoq/quiet
 
 You can access is with the local JS at
 
-- normal sources: http://localhost:8000/max/output.jsbin.com/pegizoq/quiet
-- minified: http://localhost:8000/min/output.jsbin.com/pegizoq/quiet
+http://localhost:8000/proxy/output.jsbin.com/pegizoq/quiet
 
-When accessing `min` urls make sure you run `gulp dist` with the `--fortesting`
+Note local proxy will serve minified or unminified JS based on current serve mode. When serve mode is `cdn` local proxy will serve remote JS.
```

Comment:
Rewrite as follows:

`**`Note`**`: The local proxy will serve minified or unminified JS based on the current serve mode. When the serve mode is `cdn`, the local proxy will serve remote JS.